docs/source/conf.py

```diff
@@ -19,8 +20,12 @@
 
 
 if os.environ.get("READTHEDOCS") == "True":
-    # we need to add ffmpeg to the path
     site_path = get_python_lib()
+    # bindings for pangocffi, cairocffi, pangocairocffi need to be generated
+    subprocess.run(["python", "pangocffi/ffi_build.py"], cwd=site_path)
+    subprocess.run(["python", "cairocffi/ffi_build.py"], cwd=site_path)
+    subprocess.run(["python", "pangocairocffi/ffi_build.py"], cwd=site_path)
```

Comment:
@naveen521kk how long does this take? 
@behackl what do we think of this?

Reply:
I don't know @behackl had this implementation.

Reply:
Yes, guilty as charged. I am not aware of any other way that fixes the documentation build. The required time is neglegible:

```
behackl:site-packages/ $ time python pangocffi/ffi_build.py                    [15:54:31]
python pangocffi/ffi_build.py  0,20s user 0,03s system 99% cpu 0,233 total
behackl:site-packages/ $ time python pangocairocffi/ffi_build.py               [15:54:56]
python pangocairocffi/ffi_build.py  0,23s user 0,05s system 100% cpu 0,274 total
behackl:site-packages/ $ time python cairocffi/ffi_build.py                    [15:55:12]
python cairocffi/ffi_build.py  0,22s user 0,02s system 99% cpu 0,244 total
```

Less than a second in total on my local machine, should not be too far off for RTD.

Reply:
Great, thanks!